storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -74,7 +66,7 @@ export const WithDescription: Story = {
         {args.invalid && <ErrorMessage id="error-b1">Vul uw voornaam in.</ErrorMessage>}
         <TextInput
           aria-describedby={args.invalid ? 'error-b1' : undefined}
-          aria-required="true"
+          aria-required
```

Comment:
I'm not sure this in an improvement. React turn this into `aria-required="true"`, so the end result is the same, but it sort of muddles up props and attributes imo. They are different things, and using `aria-required="true"` here makes that difference a bit more clear imo. Same for all the other instances of `aria-required`

Reply:
I've adjusted it.